flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/stream/sql/SqlITCase.scala

```diff
@@ -897,6 +897,45 @@ class SqlITCase extends StreamingWithStateTestBase {
 
     assertEquals(List(expected.toString()), StreamITCase.testResults.sorted)
   }
+
+  @Test
```

Comment:
Please add tests for table API as well. I would expect there to have this error:
```
org.apache.flink.table.api.ValidationException: Cannot translate a query with an unbounded table function call.

	at org.apache.flink.table.api.Table.getRelNode(table.scala:94)
	at org.apache.flink.table.utils.StreamTableTestUtil.printTable(TableTestBase.scala:321)
```

Reply:
Sorry, I meant the other way around. We should try to fix this for table api. By saying:
>  I would expect there to have this error:

I didn't mean that "I would like to have test asserting this validation exception", but "I think you missed testing this feature on table API and it probably will fail there with validation exception"

Reply:
@pnowojski , I think it's better to support SQL only for this time. TableAPI needs more effort.

Reply:
What's the problem with Table API here? I had a suspicion that this `ValidationException`:
```
  def getRelNode: RelNode = if (containsUnboundedUDTFCall(logicalPlan)) {
    throw new ValidationException("Cannot translate a query with an unbounded table function call.")
  } else {
    logicalPlan.toRelNode(relBuilder)
  }
```
 is being thrown mostly as a precaution, since previously there was no execution code to support it. Now (with this PR) that will not be the case anymore. What would happen if we simply removed it?